searx/static/themes/simple/src/less/style.less

```diff
@@ -540,6 +540,10 @@ article[data-vim-selected].category-social {
   h4 {
     display: none;
   }
+
+  span {
```

Comment:
Can you please do instead the selector from dalf in the issue?:
```css
#answers span {
  overflow-wrap: anywhere;
}
```

I just want to make sure that this will only get applied to spans in the `#answer` box just in case...

Reply:
nvm I did not read properly :facepalm: